app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -121,22 +121,13 @@ function buildTagsHTML(tag) {
     antifollow +
     '</h3>' +
     '<p class="grid-cell__summary truncate-at-3"></p>' +
-    '<form class="edit_follow flex items-center flex-nowrap mb-4" id="edit_follow_' +
+    '<input name="follows[][id]" id="follow_id" type="hidden" form="follows_update_form" value="' +
     tag.id +
-    '" action="/follows/' +
-    tag.id +
-    '" accept-charset="UTF-8" method="post">' +
-    '<input name="utf8" type="hidden" value="✓">' +
-    '<input type="hidden" name="_method" value="patch">' +
-    '<input type="hidden" name="authenticity_token" value="' +
-    tag.token +
     '">' +
-    '<label for="follow_points" class="fs-s flex-1 pr-2 color-base-60 align-right whitespace-nowrap">Follow weight:</label>' +
-    '<input step="any" class="crayons-textfield flex-1 fs-s" required="required" type="number" style="max-width:90px" value="' +
+    '<label for="follow_points" class="fs-s flex-1 pr-2 color-base-60 align-left whitespace-nowrap">Follow weight:</label>' +
+    '<input step="any" class="crayons-textfield flex-1 fs-s" required="required" type="number" form="follows_update_form" value="' +
     tag.points +
-    '" name="follow[points]" id="follow_points">' +
-    '<button type="submit" class="crayons-btn crayons-btn--ghost crayons-btn--s" name="commit">Save</button>' +
-    '</form>' +
+    '" name="follow[][points]" id="follow_points" onchange="addChanged(this)">' +
```

Comment:
Also here -- this seems to work as `follow` or `follows` but let's be consistent and keep it as one or the other. :)